src/EncryptionBehavior.php

```diff
@@ -213,15 +229,16 @@ protected function createRealColumnName(string $columnName, string $tableName):
 
     /**
      * @param string $columnName
-     * @param array $searchableColumnNames
+     * @param array<string> $searchableColumnNames
      *
-     * @return boolean
+     * @return bool
```

Comment:
I see you changed **boolean** to **bool** but not all places:
https://github.com/spryker/propel-encryption-behavior/blob/8d415afb613a385d9887b5aa5fac645f4c8286bb/src/EncryptionBehavior.php#L272,
https://github.com/spryker/propel-encryption-behavior/blob/8d415afb613a385d9887b5aa5fac645f4c8286bb/src/EncryptionBehavior.php#L281,
https://github.com/spryker/propel-encryption-behavior/blob/8d415afb613a385d9887b5aa5fac645f4c8286bb/src/EncryptionBehavior.php#L121,
etc...

Reply:
Does that mean the Code sniffer doesnt see those cases for some reason?

Reply:
Ah, you didn't add cs run into CI

Reply:
Alao phpstan please

Reply:
See decimal-object for reference

Reply:
https://github.com/spryker/decimal-object

Reply:
@dereuromark Do you know why this error happened? 
https://github.com/spryker/propel-encryption-behavior/runs/4775149065?check_suite_focus=true
I faced this error locally but reinstalling composer packages solves this error

Reply:
Check how the ruleset should be added, other repos use it, so does project nonsplit